openreview/conference/templates/programchairWebfield.js

```diff
@@ -254,6 +281,102 @@ var displayHeader = function() {
   Webfield.ui.tabPanel(tabs);
 };
 
+var displayConfiguration = function(requestForm, invitations) {
+
+  var formatPeriod = function(invitation) {
+    var start;
+    var end;
+    var afterStart = true;
+    var beforeEnd = true;
+    var now = Date.now();
+    if (invitation.cdate) {
+      var date = new Date(invitation.cdate);
+      start =  date.toLocaleDateString('en-GB', { hour: 'numeric', minute: 'numeric', day: '2-digit', month: 'short', year: 'numeric', timeZoneName: 'long'});
+      afterStart = now > invitation.cdate;
+    }
+    if (invitation.duedate) {
+      var date = new Date(invitation.duedate);
+      end =  date.toLocaleDateString('en-GB', { hour: 'numeric', minute: 'numeric', day: '2-digit', month: 'short', year: 'numeric', timeZoneName: 'long'});
+      beforeEnd = now < invitation.duedate;
+    }
+
+    var currentPeriod = afterStart && beforeEnd;
+    var periodString = start ? 'from <i>' + start + '</i> ' : 'open ';
+    if (end) {
+      periodString = periodString + 'until <i>' + end + '</i>';
+    } else {
+      periodString = periodString + 'no deadline';
+    }
+
+    return periodString;
+  }
+  var invitationMap = {};
+
+  invitations.forEach(function(invitation) {
+    invitationMap[invitation.id] = invitation;
+  });
+
+  var container = '#venue-configuration';
+  var html = '<div id="container"></br>'
+
+  if (requestForm) {
+    html = html +  '<a href="/forum?id=' + requestForm.id + '">Requested configuration</a><span>: ' + getConfigurationDescription(requestForm) + '</span></br></br>';
```

Comment:
Could you provide an example? I'm working on all your suggestions and I can do the change.

Reply:
Actually, I don't really know if a ul would really be better in this case. It would be better to wrap the Requested configuration section in a p tag rather than use br's though. Also `html += ...` is a pretty common pattern that we use other places.